src/scripts/main.js

```diff
@@ -1,3 +1,28 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('table');
+const tbody = table.querySelector('tbody');
+const headers = table.querySelectorAll('th'); // Виправлено на querySelectorAll
+
+headers.forEach((th, index) => {
+  th.addEventListener('click', () => {
+    const rows = Array.from(tbody.querySelectorAll('tr'));
+    const isNumberColumn = index === 2 || index === 3;
+
+    rows.sort((a, b) => {
+      const aText = a.cells[index].textContent.trim();
+      const bText = b.cells[index].textContent.trim();
+
+      if (isNumberColumn) {
+        const aValue = parseFloat(aText.replace(/[$,]/g, ''));
+        const bValue = parseFloat(bText.replace(/[$,]/g, ''));
```

Comment:
Ensure that the data in the numeric columns is consistently formatted. If there are any non-numeric values, `parseFloat` will return `NaN`, which could affect sorting. Consider adding error handling or data validation to manage such cases.